docs/_templates/sidebar/brand.html

```diff
@@ -16,26 +16,33 @@
   {%- endif %}
   {% endblock brand_content %}
 </a>
+<script>
+  function setPrefix(prefix){
+    window.sessionStorage.setItem("version-select-index", document.getElementsByClassName("version-select")[0].selectedIndex);
```

Comment:
Do we have to use session storage?
Can't we just open the versioned page ?

Reply:
Ah this is to save the state of the selector between pages. Otherwise, when you switch document versions, you will be brought to the correct page but the selector will default back to `latest`

Reply:
By having the session state, we save the item they selected before they changed pages so the selector is consistent